File: Projeto Mobile - Forms/flutter_mobile/lib/evento.dart
import 'package:flutter/material.dart';

class Evento {
  String _id = '';
  String _nome = '';
  String _data = '';
  String _hora = '';
  String _palestrante = '';
  int _maxAlunos = 0;

  String get id => _id;
  set id(String value) => _id = value;

  String get nome => _nome;
  set nome(String value) => _nome = value;

  String get data => _data;
  set data(String value) => _data = value;

  String get hora => _hora;
  set hora(String value) => _hora = value;

  String get palestrante => _palestrante;
  set palestrante(String value) => _palestrante = value;

  int get maxAlunos => _maxAlunos;
  set maxAlunos(int value) => _maxAlunos = value;
}

File: GenesisAPI/src/main/java/com/example/demo/model/Evento.java
package com.example.demo.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
@Table
public class Evento {
    @Id
    private String id;
    @Column
    private String nome;
    @Column
    private String data;
    @Column
    private String hora;
    @Column
    private String palestrante;
    @Column
    private String maxAlunos;
}

